src/main/java/MysqlDatabase.java

```diff
@@ -0,0 +1,96 @@
+import org.eclipse.jgit.revwalk.DepthWalk;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MysqlDatabase {
+    /**
+     * Creates a Connection object and connects it to localhost mysql server
+     * @return Connection object
+     */
+    public static Connection connectToDB (){
+        try{
+            //Try getting an object from class com.mysql.jdbc.Driver
+            try {
+                Class.forName("com.mysql.cj.jdbc.Driver");
+            } catch (ClassNotFoundException classNotFound) {
+                classNotFound.printStackTrace();
+            }
+            Connection connection = null;
+
+            //Try connecting to database 'test' with username=root and password=root.
+            try {
+                connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/test?autoReconnect=true&useSSL=false","root","root");
+            } catch (SQLException throwables) {
+                throwables.printStackTrace();
+            }
+            return connection;
+        } catch(Exception e) {
+            System.out.println(e);
+        }
+        return null;
+    }
+
+    /**
+     * Inserts the given CommitStructure object into the database
+     * @param commit The CommitStructure Object with set values for inserting into database
+     * @return Boolean that signals if the insert was successfull or not
+     */
+    public static boolean insertCommitToDatabase(CommitStructure commit) throws SQLException {
+        Connection connection = connectToDB();
+        if(connection!=null){
+            //Use prepared statements for security purposes
+            PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO commit VALUES(?,?,?,?,?)");
+            preparedStatement.setString(1, commit.getCommitID());
+            preparedStatement.setString(2, commit.getBuildDate());
+            preparedStatement.setBoolean(3, commit.isTestResult());
+            preparedStatement.setBoolean(4, commit.isBuildResult());
+            preparedStatement.setString(5, commit.getBuildLogs());
+
+            int result = preparedStatement.executeUpdate();
+            preparedStatement.close();
+            assert connection != null;
+            connection.close();
+            //Result > 0 => successfully inserted into database
+            return (result>0);
+        }
+        else{
+            System.out.println("Connection is null!");
+        }
+        return false;
+    }
+
+    /**
+     * Selects all rows from database
+     * @return Returns a list with all of the rows read as CommitStructure objects
+     */
+    public static List<CommitStructure> selectAllCommits() throws SQLException {
```

Comment:
If connection is used as parameter, we can test these better

Reply:
Fixed in commit https://github.com/DD2480-2021-group-22/dd2480-lab-2/pull/24/commits/0aa78b81fc4b991068de267a30ba058e373b13e4